sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewViewModel.groovy

```diff
@@ -38,6 +38,8 @@ class ProjectOverviewViewModel {
         projectResourceService.iterator().each { Project project ->
             addProject(project)
         }
+        Collections.sort(projectOverviews, new LastChangedComparator(SortOrder.DESCENDING))
+        
```

Comment:
This seems not to work. I think you need to import the SortOrder. I tested it locally and I got an error

Reply:
should be fixed now